### PR TITLE
Common - Fix unloading person on non-local vehicles

### PR DESCRIPTION
**Pull request by jokoho48**
**When merged this pull request will:**
- Fixes issues with not being able to unload players from vehicles that are remote to the unloaded player

### CODE REVIEW

File: addons/common/XEH_postInit.sqf [hunk 1]
```diff
@@ -133,6 +133,11 @@ if (isServer) then {
     _this lock (_this getVariable [QGVAR(lockStatus), locked _this]);
 }] call CBA_fnc_addEventHandler;
 
+[QGVAR(EjectVehicle), {
```
Comment (jonpas) [inline addons/common/XEH_postInit.sqf:136]:
```suggestion
[QGVAR(ejectVehicle), {
```

Comment (jokoho48) [reply addons/common/XEH_postInit.sqf:136]: sorry i am still testing and noticed that this still does not bring the wanted effect

**Comment by jonpas** [comment]
Is this still WIP?

**Comment by jokoho48** [comment]
> 
> 
> Is this still WIP?

no. else i would have a WIP tag or WIP in the title 😁

**Comment by jonpas** [comment]
Can you explain how this fixes it?

**Comment by jokoho48** [comment]
> 
> 
> Can you explain how this fixes it?

the generall issue is that `_unit action ["Eject", _vehicle];` only works if the _vehicle AND _unit are local. moveOut works if _vehicle is not local. so we use the old code with `_unit action ["Eject", _vehicle];` if the _vehicle is local and do moveOut if _vehicle is not local

**Review by jonpas — APPROVED** [review]

**Comment by jonpas** [comment]
Someone else review as well!

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**